lib/media/presentation_timeline.js

```diff
@@ -361,6 +361,13 @@ shaka.media.PresentationTimeline = class {
   getSegmentAvailabilityEnd() {
```

Comment:
The JSDoc for the return value, above, says "Aways returns the presentation's duration for video-on-demand." That seems to be pretty out-of-date, now.

Reply:
Fixed.  Thanks!